packages/compartment-mapper/src/policy.js

```diff
@@ -171,21 +175,27 @@ export const getPolicyForPackage = (namingKit, policy) => {
   }
 };
 
+/**
+ * Get list of globals from package policy
+ * @param {import('./types.js').PackagePolicy} [packagePolicy]
+ * @returns {Array<string>}
+ */
 const getGlobalsList = packagePolicy => {
-  if (!packagePolicy.globals) {
+  if (!packagePolicy?.globals) {
     return [];
   }
-  return entries(packagePolicy.globals)
+  return entries(packagePolicy?.globals)
```

Comment:
(should) avoid the `?` here, there's an early return that handles it already, and passing `null` to `entries` would throw anyway.